err_stash.py

```diff
@@ -42,22 +47,125 @@ def fetch_repo_commits(self, project, slug, until, since):
         return self._stash.projects[project].repos[slug].commits(until, since)
 
 
+class GithubAPI:
+    """
+    Access to the pygithub API.
+    """
+    def __init__(self, login_or_token=None, password=None, organizations=[]):
```

Comment:
Just out of curiosity, are you suggesting this to prevent doing that "static" set like this?
```
class Foo():
  def __init(a=[]):
    self.a = a # and all instances of Foo now share the same `a` field
```

Reply:
That's a common gotcha, see "Mutable Default Arguments" [here](https://8thlight.com/blog/shibani-mookerjee/2019/05/07/some-common-gotchas-in-python.html)

Reply:
Done